content/en/docs/tasks/job/automated-tasks-with-cron-jobs.md

```diff
@@ -26,20 +26,16 @@ Therefore, jobs should be idempotent.
 For more limitations, see [CronJobs](/docs/concepts/workloads/controllers/cron-jobs).
 
 
-
 ## {{% heading "prerequisites" %}}
 
-
 * {{< include "task-tutorial-prereqs.md" >}}
 
-
-
 <!-- steps -->
 
-## Creating a Cron Job
+## Creating a Cron Job {#creating-a-cronjob}
```

Comment:
I don't think "cron job" is a good English term. Cron is named after Greek word “Chronos” that is used for time. Cron itself is a jargon. I'd suggest we call CronJobs CronJobs rather than "cron jobs".

Reply:
Do we need changed the `cron job` in body text to `CronJob` when all the headlines show `CronJob` now?

Reply:
@tengqm As far as I know, "cron" is the right word and the best way to put this out would be in "cron job" which is how I have seen it being used elsewhere too.

The [dictionary meaning](https://www.google.com/search?q=meaning+of+cron) of cron seconds this and an example of using this in a sentence as shown in the default Google Dictionary is:

> "a cron job"

Also, here's a [Google Cloud](https://cloud.google.com/scheduler/docs/creating) documentation page using "cron job".